uint32 speakerId
geometry_msgs/Point position